airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -20,9 +27,12 @@
 class TwilioStream(HttpStream, ABC):
     url_base = TWILIO_API_URL_BASE
     primary_key = "sid"
-    page_size = 100
     transformer: TypeTransformer = TypeTransformer(TransformConfig.DefaultSchemaNormalization | TransformConfig.CustomSchemaNormalization)
-
+    
+    def __init__(self, pagesize: int = 50, **kwargs):
+        super().__init__(**kwargs)
+        self._page_size = pagesize
```

Comment:
```suggestion
    def __init__(self, page_size: int, **kwargs):
        super().__init__(**kwargs)
        self._page_size = page_size
```
I don't think you need to declare a default because the source explicitly defines the page_size attribute. Please rename to `page_size` for consistency.